app/routes.js

```diff
@@ -1414,3 +909,7 @@ router.get('/prototype-admin/show-data', (req, res, next) => {
   next()
 })
 
+/* Use the routes file in pages for adding and editing questions */
+router.use('/pages', require('./views/form-designer/pages/\_routes'))
```

Comment:
It should be sufficient to just require the extra routes file

```suggestion
require('./views/form-designer/pages/_routes')
```